site/src/components/series/errorBar.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: component
+title: Error Bar Series
+component: series/errorBar.js
+tags:
+  - frontpage
+namespace: series
+
+example-code: |
+  var point = fc.series.point()
+      .xScale(xScale)
+      .yScale(yScale);
+  
+  container.append('g')
+      .datum(data)
+      .call(point);
+  
+  //Generating the error information for the data
+  data.forEach(function(d) {
+      d.yUpError = Math.random() * 2;
+      d.yDownError = Math.random() * 2;
+  });
+
+  var eBar = fc.series.errorBar()
+      .xScale(xScale)
+      .yScale(yScale)
+      .yLow(function(d,i) {return d.close - d.yDownError;})
+      .yHigh(function(d,i) {return d.close + d.yUpError;})
+      .yValue(function(d,i) {return d.close;});
```

Comment:
I think the yLow etc. lines wouldn't pass eslint due to the whitespace with the `{`, but these aren't linted because they're in the md. In a more general sense is it possible to lint these files? And should we try to make these sections eslint compliant?